charts/rest-devicemanager/templates/middleware-headers.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: traefik.containo.us/v1alpha1
```

Comment:
I checked with the dev team, I'll keep it because current application accept Cors.

I'll note that I can delete this middleware when this application not using local Cors.

Thank you